WordPress/gutenberg
# [Create Block] Adding a `--no-plugin` flag to scaffold only block files.

## What?

This PR introduces the `--no-plugin` flag to allow users to use the package to scaffold only block files.

## Why?
Currently, there is no way to scaffold additional blocks once the plugin has been created. Being able to create subsequent blocks quickly would be very helpful.

## How?
When the `--no-plugin` flag is added, the script skips any plugin scaffolding and only outputs the files listed in the `blockTemplatesPath` ( just the block files ) into `folderName` as defined by the template with the default being `./src`. It should be noted that the script is not "aware" of its location. Calling `npx @wordpress/create-block block-name --no-plugin` will create a new block inside the `folderName`directory where it's called

The slug passed is used as the directory and block name any other flags not related to package.json items should function as expected for the block, i.e namespace. 

## Testing Instructions
1. From the root of `packages/create-block`, run `node index.js test ` to create a new plugin with a single block
2. `cd` into the `test` directory
3.  Run `node ../index.js my-test-block --no-plugin` to scaffold a new block
4. Confirm that the `src` directory has the existing block files and a new directory called `my-test-block` that contains new block files
5. Run `npm start` to confirm that the build works

## Screenshots or screencast
